qt5-apps/qt-creator/DETAILS

```diff
@@ -1,18 +1,18 @@
           MODULE=qt-creator
-         VERSION=opensource-src-4.0.2
+            VERS=4.0.3
+         VERSION=opensource-src-$VERS
```

Comment:
This looks better to me;

          MODULE=qt-creator
         VERSION=4.0.3
          SOURCE=$MODULE-opensource-src-$VERSION.tar.gz
      SOURCE_URL=http://download.qt-project.org/official_releases/qtcreator/${VERSION%.*}/$VERSION/